assemblers/io.weldr.qcow2

```diff
@@ -79,5 +78,6 @@ def main(tree, input_dir, options):
 
 if __name__ == '__main__':
     args = json.load(sys.stdin)
-    r = main(args["tree"], args["input_dir"], args["options"])
+    print(f"Args: {args}")
```

Comment:
Please remove the `print` statement here.

Reply:
I think we should introduce some verbose mode, that would print this. Anyway it's gone for now.